apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -414,61 +466,106 @@ class TopInstructionsCSP extends Component {
       this.state.tabSelected === TabType.COMMENTS &&
       this.state.teacherViewingStudentWork;
 
+    if (
+      hidden ||
+      (!shortInstructions && !longInstructions && !hasContainedLevels)
+    ) {
+      return <div />;
```

Comment:
could we return `null` here instead? then nothing would be rendered rather than an empty `div`

Reply:
I can try this... this was the way it was set up before so I don't know if there was a reason

Reply:
eh, if that's the existing behavior, let's leave it for now